instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/log/ConsentPendingGrantedLogsTest.kt

```diff
@@ -33,7 +32,7 @@ internal class ConsentPendingGrantedLogsTest : LogsTest() {
     @Test
     fun verifyActivityLogs() {
         // update the tracking consent
-        Datadog.setTrackingConsent(TrackingConsent.GRANTED)
+        Datadog.getInstance()?.setTrackingConsent(TrackingConsent.GRANTED)
```

Comment:
I'm wondering if we can also have some non-null variant of getting the instance, because for example in nightly-tests we have to use `!!`, which is not very nice.

To me it is the same situation as getting the `Activity` in Android, where `getActivity` and `requireActivity` exist (which looks to me as a hacky workaround).

Reply:
Yes, I'm not a fan of such API either. I'm just curious how we can reduce the number of nullable states in our codebase.